applications/GeoMechanicsApplication/tests/test_GeoMechanicsApplication.py

```diff
@@ -91,6 +93,7 @@ def AssembleTestSuites():
                         KratosGeoMechanicsStrainMeasureTests,
                         KratosGeoMechanicsSetMultipleMovingLoadProcessTests,
                         KratosGeoMechanicsRotationWithMovingLoadTests,
+                        KratosGeoMechanicsCPhiReductionProcess,
```

Comment:
Doubting a bit if this should be in small or nightly test cases, what do you think?

Reply:
Yes, it takes 15 seconds on Github. Moved to nighty